lsp/.eslintrc.js

```diff
@@ -0,0 +1,22 @@
+/**@type {import('eslint').Linter.Config} */
+// eslint-disable-next-line no-undef
+module.exports = {
```

Comment:
Unrelated to this PR: it'd be nice to standardize some of these TS style/linting configs in a shared module.